modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/FolderCopyEntryMVCRenderCommand.java

```diff
@@ -34,11 +34,12 @@
 		"javax.portlet.name=" + DLPortletKeys.DOCUMENT_LIBRARY,
 		"javax.portlet.name=" + DLPortletKeys.DOCUMENT_LIBRARY_ADMIN,
 		"javax.portlet.name=" + DLPortletKeys.MEDIA_GALLERY_DISPLAY,
-		"mvc.command.name=/document_library/copy_folder"
+		"mvc.command.name=/document_library/folder_copy_entry"
 	},
 	service = MVCRenderCommand.class
 )
-public class CopyFolderMVCRenderCommand extends BaseFolderMVCRenderCommand {
+public class FolderCopyEntryMVCRenderCommand
+	extends BaseFolderMVCRenderCommand {
```

Comment:
Don't we have now here (and in other render commands) the same wording problem mentioned by Brian?

> AbcXyz extends BaseXyz

Reply:
You're right, but that's the only way for the command names to match for both action and render commands. It would be super confusing to have the command name change depending on the request type.

Reply:
I've updated the PR description to highlight this.

Reply:
So maybe these changes are not worth, if we make changes to accomplish with a wording in one side but stop accomplishing with wording in another side... 😅